scripts/diff_schema.sh

```diff
@@ -53,7 +53,7 @@ echo
 RESULTS="./schema_diffs-$LEFT_TAG-$RIGHT_TAG.txt"
 
 # Remove schemas on exit
-trap 'rm -rf ./*/*/"schema-$LEFT_TAG" ./*/*/"schema-$RIGHT_TAG"' 0
+#trap 'rm -rf ./*/*/"schema-$LEFT_TAG" ./*/*/"schema-$RIGHT_TAG"' 0
```

Comment:
Just remove that part? Why comment?

Reply:
This was commented during tests. Good catch.